### ISSUE TITLE
Nbitcoin Qbitninja.client testnet (Spend your coin)

### ISSUE THREAD

**Issue by jole95** [ProgrammingBlockchain/ProgrammingBlockchain#135]
transactionResponse is null 😓😣😞. why?

**Comment by nopara73** [comment]
You can also use Wasabi's broadcast post instead: https://wasabiwallet.io/swagger/

https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi.Backend/Controllers/BlockchainController.cs#L314

**Comment by NicolasDorier** [comment]
@jole95 I think qbitninja testnet is down. I don't have time to maintain it. :( As @nopara73  say, you can try by sending manually or via wasabi

**Comment by jole95** [comment]
@nopara73  I'll try. Thanks a lot 🙂
@NicolasDorier It is a shame because there is well written documentacion and simple code for me beginner☹

**Comment by NicolasDorier** [comment]
@jole95 yeah sorry about that. I'll try to resurrect it later on. Or update the documentation to use more modern approach.

**Comment by jole95** [comment]
@NicolasDorier Please help me 😃

**Comment by kazute** [comment]
Is there any tutorial to use the wasabi broadcaster over testnet rather than qbitninja?

**Comment by nopara73** [comment]
I'm afraid there isn't. Maybe go with Blockstream's tx broadcasting, they have documentation: https://github.com/Blockstream/esplora/blob/master/API.md#post-tx
